sdk/attestation/Azure.Security.Attestation/tests/AttestationClientTestEnvironment.cs

```diff
@@ -0,0 +1,166 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.IO;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+
+
+// licy":"CnZlcnNpb249MS4 => version=1.
+// licy":"DQp2ZXJzaW9uPTE => version=1
+
+namespace Azure.Security.Attestation.Tests
+{
+    public class AttestationClientTestEnvironment : TestEnvironment
+    {
+        public AttestationClientTestEnvironment() : base("attestation")
+        {
+        }
+
+        public string IsolatedAttestationUrl => GetRecordedVariable("ISOLATED_ATTESTATION_URL");
+        public string AadAttestationUrl => GetRecordedVariable("AAD_ATTESTATION_URL");
+
+        public X509Certificate2 PolicyCertificate0 => new X509Certificate2(Convert.FromBase64String(GetRecordedVariable("policySigningCertificate0")));
+        public X509Certificate2 PolicyCertificate1 => new X509Certificate2(Convert.FromBase64String(GetRecordedVariable("policySigningCertificate1")));
+        public X509Certificate2 PolicyCertificate2 => new X509Certificate2(Convert.FromBase64String(GetRecordedVariable("policySigningCertificate2")));
+
+        public RSA PolicySigningKey0 => GetRSACryptoServiceProvider("serializedPolicySigningKey0");
+        public RSA PolicySigningKey1 => GetRSACryptoServiceProvider("serializedPolicySigningKey1");
+        public RSA PolicySigningKey2 => GetRSACryptoServiceProvider("serializedPolicySigningKey2");
+
+        // Policy management keys.
+        public X509Certificate2 PolicyManagementCertificate => new X509Certificate2(Convert.FromBase64String(GetRecordedVariable("isolatedSigningCertificate")));
+        public RSA PolicyManagementKey => GetRSACryptoServiceProvider("serializedIsolatedSigningKey");
+
+
+        public string SharedEusTest => "https://sharedeus.eus.test.attest.azure.net";
+        public string SharedUkSouth => "https://shareduks.uks.test.attest.azure.net";
+
+        public string ActiveDirectoryTenantId => GetRecordedVariable("TENANT_ID");
```

Comment:
These are available as properties on the base TestEnvironment - TenantId, ClientId, and ClientSecret.

Reply:
Thanks - I had written these before I found them.